fixtures/cron/test-cron.handler.ts

```diff
@@ -1,13 +1,11 @@
 import { CronHandler } from '../../handlers';
-import { Schedule } from 'aws-cdk-lib/aws-events';
 
 export const handler = CronHandler(
 	{
 		name: 'queue-pulls',
-		schedule: Schedule.cron({
-			hour: '4',
-			minute: '0',
-		}),
```

Comment:
Dunno if aws updated their package or something, but this no longer works and fails the test. Throwing it in manually as below works great.